actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -1255,11 +1255,8 @@ public boolean canActivateActor(final String interfaceName)
     {
         if (getState() != NodeCapabilities.NodeState.RUNNING)
         {
-            // todo, improve this
-            if (hosting.getServerNodes().size() > 1)
```

Comment:
probably here to avoid a deadlock trying to call actors after the node shutdown.